### DISCUSSION TITLE
<stal> highlighting party members

### DISCUSSION THREAD

**Discussion by jwest384** [tirem/XIUI#5] [Ideas]
Hello.  When using <stal> to cycle through party members to cast of them, it is not currently highlighting them in the party list.  Is this a feature that will be added at a later date?

**Comment by tirem** [comment]
Is it only "stal" that you aren't noticing working or any "st" ? If you grab the latest version you should notice the little arrow move around when you have a subtarget. If you have no target and activate a subtarget both the highlight and the arrow will move together.

**Reply by Poklin5634** [reply]
what about stpt, my friend was trying it and it didn't work. but this was yesterday i think so idk if you added it also.

**Reply by jwest384** [reply]
> Is it only "stal" that you aren't noticing working or any "st" ? If you grab the latest version you should notice the little arrow move around when you have a subtarget. If you have no target and activate a subtarget both the highlight and the arrow will move together.

It seems to only be < stal>.  < st> and < stpc> are highlighting correctly.  Here is a screenshot.  I used <stal> and you can see myself targeted in the normal party list at bottom-right but I'm not highlighted in the ConsolidatedUI party list.

![stal](https://user-images.githubusercontent.com/77185044/212493280-43eaf86a-b0f4-4164-a5dc-08319246bc9e.png)

**Reply by tirem** [reply]
I see you're using the non latest build. Could you update to the latest version and lmk if it's still an issue?

**Reply by tirem** [reply]
I can look into "stal" as well though ^^ it's odd that would behave differently.

**Reply by tirem** [reply]
I found the issue. you can expect a fix sometime this weekend!

**Comment by Poklin5634** [comment]
@tirem idk if you saw what i said but <stpt> isn't working as of now

**Comment by tirem** [comment]
This should be fixed in the newest version :)

**Reply by jwest384** [reply]
Can confirm this is now working!  Thank you so much for this amazing UI!
